riff-raff/app/deployment/Deployments.scala

```diff
@@ -36,8 +35,6 @@ class Deployments(deploymentEngine: DeploymentEngine,
       requestedParams.build.projectName, requestedParams.stage.name, requestSource)
     if (restrictionsPreventingDeploy.nonEmpty) {
       Left(Error(s"Unable to queue deploy as restrictions are currently in place: ${restrictionsPreventingDeploy.map(r => s"${r.fullName}: ${r.note}").mkString("; ")}"))
-    } else if (enableQueueingSwitch.isSwitchedOff) {
```

Comment:
(I have not recreated the enableQueueing switch as was not convinced of the value, but we can re-add if it's more useful than I realise.)

Reply:
I don't remember this being used in my time on DevX so I think it's OK to remove it.